doc/dev/release.md

```diff
@@ -306,11 +307,24 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**Note**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
```

Comment:
```suggestion
**Note:**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
```